.travis.yml

```diff
@@ -6,29 +6,28 @@ go:
     - 1.6
     
 env:
-    - BUILDER_VERSION=0.11.0
+    global:
+        - BUILDER_VERSION=0.11.0
+    matrix:
+        - TEST_SUITE=unit
+        - TEST_SUITE=integration
 
 cache:
     directories:
-    - ./builder-cache
+        - ./builder-cache
 
 notifications:
     email: false
 
 before_install:
-- cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
-- export PATH=$PATH:$PWD
-- go get -u github.com/golang/lint/golint
+    - openssl aes-256-cbc -K $encrypted_314659c9f31c_key -iv $encrypted_314659c9f31c_iv -in inago-integration-test.pem.enc -out ./inago-integration-test.pem -d
+    - cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
+    - chmod +x ./builder
+    - export PATH=$PATH:$PWD
+    - go get -u github.com/golang/lint/golint
 
 script:
-- make lint
-- make ci-test
-- make ci-build
-- ./inagoctl
+    - ./ci.sh
```

Comment:
didn't you opt against this in the past? Whats the reason for doing this with the `matrix` option above now?